src/PowerShellEditorServices/Services/PowerShellContext/PowerShellContextService.cs

```diff
@@ -423,19 +419,14 @@ public static Runspace CreateRunspace(PSHost psHost)
         /// Imports the PowerShellEditorServices.Commands module into
         /// the runspace.  This method will be moved somewhere else soon.
         /// </summary>
-        /// <param name="moduleBasePath"></param>
         /// <returns></returns>
-        public Task ImportCommandsModuleAsync(string moduleBasePath)
+        public Task ImportCommandsModuleAsync()
         {
-            PSCommand importCommand = new PSCommand();
-            importCommand
+            PSCommand importCommand = new PSCommand()
                 .AddCommand("Import-Module")
-                .AddArgument(
-                    Path.Combine(
-                        moduleBasePath,
-                        "PowerShellEditorServices.Commands.psd1"));
+                .AddArgument(s_commandsModulePath);
 
-            return this.ExecuteCommandAsync<PSObject>(importCommand, false, false);
+            return this.ExecuteCommandAsync<PSObject>(importCommand, sendOutputToHost: false, sendErrorToHost: false);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this default value assigned to parameter 'sendOutputToHost'.](https://app.codacy.com/app/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=4579423)